ui/src/style/pdf-viewer.less

```diff
@@ -15,4 +15,14 @@
   */
 .linkAnnotation a[href^="#cite"] {
   display: none;
+}
+
+/** 
+  * Consistent highlight between default pdf.js highlight for tables, figures, etc. 
+  * and custom purple highlight for citation annotations.
+  */
+.linkAnnotation a:not([href^="#cite"]):hover {
+  background: rgb(0, 0, 255);
```

Comment:
@jocelynzsun Can this use variables for background, opacity, and box-shadow, so that colors and opacity of annotations only need to be updated in one place when we change them? The steps would be:

1. Define new variables in `variables.less`
2. Refer to those variables here, and in the styles in `annotation.less`

Reply:
I don't think any of these variables are used in `annotation.less` at this point?

Reply:
Can we use the same variables and CSS properties to style both these link annotations and our annotations? I'm looking at these styles from `annotation.less`. It would be great if they could both use the same properties (e.g., either `background` or `background-color`) and had the same hover and click behavior too!

https://github.com/allenai/scholar-reader/blob/5019fa9f446bb2062d9f37f61630c1ecb78d09d0/ui/src/style/annotation.less#L16-L41